source/frontend/src/components/common/mapFSM/machineDefinition/mapMachine.ts

```diff
@@ -224,6 +224,27 @@ const sideBarStates = {
   },
 };
 
+const advancedFilterSideBarStates = {
+  initial: 'fullScreen',
```

Comment:
If this is the state of the filter, should it be initially closed?

Reply:
I modelled this after the "regular" sidebar - which starts in the "fullScreen" mode == closed

Reply:
I actually like the idea of having the controls of the map as a state given only one can be valid at a time. Having the states being 
none,
advancedFilter,
layers.